include/xrpl/protocol/UintTypes.h

```diff
@@ -129,6 +129,12 @@ struct hash<ripple::Directory> : ripple::Directory::hasher
     explicit hash() = default;
 };
 
+template <>
+struct hash<ripple::uint256> : ripple::uint256::hasher
```

Comment:
What is the objective of this function?

Reply:
It allows `ripple::uint256` type to be used with std library functionalities that require hashing. It is actually strange that it was not implemented  before